### ISSUE TITLE
Adding/Loading hyperlink pictures (URLs) from web/external

### ISSUE THREAD

**Issue by enceladus3** [python-openxml/python-docx#1002]
Is there a way I can add pictures/URLs from the web to a docx file?

Flow:

1. Copy image URL
2. Insert somewhere inside the document
3. Have it loaded once you open the document 
- The image hast to be loaded from the URL once the document is opened (not downloaded locally when the file is generated)

**Comment by scanny** [comment]
No. The web works that way, but PowerPoint doesn't. You can put a hyperlink to something, but the user would have to cliick on it and then view the resource in their browser. All the resources of a .pptx file are included inside the .pptx file.

**Comment by enceladus3** [comment]
@scanny This is the use-case I was talking about: https://stackoverflow.com/questions/47389192/loading-an-online-image-in-a-word-document

This already has been covered using _Open XML SDK_ so I thought that it can also be implemented using Python for a .docx file.

**Comment by scanny** [comment]
Huh, interesting. It would appear I am mistaken. I've never seen this but apparently you _can_ make a picture shape that refers to an external image and then just provide the URL to it.

There is no API support for this, so you'd be on your own for how to implement it. I don't see a way to do this from the PowerPoint UI, so that would be something to investigate. Maybe it would make sense to patch the XML of a presentation to see if you can get it to work. The first step would be having a working example that demonstrates the XML configuration and the PowerPoint application behavior.

**Comment by abubelinha** [comment]
I recall in old versions of Word you could either embed or link images in a .doc file (from local disk or from another remote machine):
- <http://www.prc.dk/user-friendly-manuals/ufm/tip-9907.htm>
- <https://word.tips.net/T000396_Understanding_Graphic_Linking.html>
- <http://wordfaqs.ssbarnhill.com/RedX.htm>

So what about images linked from the local filesystem? (outside the .docx zipped file)  
I wonder if there is any example of doing that with python-docx, or it has the same problem you were talking about.
